shared/contexts/ThemeContext.js

```diff
@@ -24,18 +24,30 @@ function addLinkNode() {
   document.head.appendChild(newLink);
 }
 
+const getEditorTheme = theme => {
+  switch (theme) {
+    case 'dark':
+      return 'vs-dark';
+    case 'hcb':
+      return 'hc-black';
+    default:
+      return 'vs';
+  }
+};
 export const ThemeProvider = ({ children, env }) => {
   const [theme, setTheme] = useState(getInitialTheme());
+  const [editorTheme, setEditorTheme] = useState(getEditorTheme(theme));
 
   useEffect(() => {
     if (typeof env.PUBLIC_URL === 'undefined') return;
     applyThemeToLinkNode(theme, env.PUBLIC_URL);
     localStorage.setItem('busola.theme', theme);
     LuigiClient.sendCustomMessage({ id: 'busola.theme', name: theme });
+    setEditorTheme(getEditorTheme(theme));
   }, [theme]);
 
   return (
-    <ThemeContext.Provider value={{ theme, setTheme }}>
+    <ThemeContext.Provider value={{ theme, editorTheme, setTheme }}>
```

Comment:
I think we can save some renders here by not using a dedicated state for the `editorTheme`
```suggestion
    <ThemeContext.Provider value={{ theme, editorTheme: getEditorTheme(theme), setTheme }}>
```